renv.lock

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
IMHO it would be good to not do all of this renv bumping as part of this work. We might want to bump our pins like this at some point, but the churn in these files is distraction from what we're pushing here

Reply:
Yes, fair — I had to do this to get the repo running on my machine but should probably cherrypick it into a different PR.

Reply:
https://github.com/rstudio/connectapi/pull/285